sycamore/transforms/schema.py

```diff
@@ -0,0 +1,180 @@
+from abc import ABC, abstractmethod
```

Comment:
Super minor, but can we call this file something like `extract_schema.py` rather than just `schema.py`. Schema is such an overloaded term it could mean any number of things.

Reply:
done 👍